Documentation/guides/Extending LiveCode.md

```diff
@@ -487,19 +688,1248 @@ The rectangle of each child widget is controlled using the following syntax:
 		set the rectangle of mClock to rectangle [ 0, 50, my width, my height ]
 	end handler
 
-Finally, messages posted by child widgets can be handled in the direct parent by handling 
-the appropriate message (prepending 'On'). For example, the selector widget posts 
-*optionChanged* when one of its numbers is selected. This is handled by this composed 
-widget example in an *OnOptionChanged* handler.
+Finally, messages posted by child widgets can be handled in the direct 
+parent by handling the appropriate message (prepending 'On'). For 
+example, the selector widget posts *optionChanged* when one of its 
+numbers is selected. This is handled by this composed widget example in 
+an *OnOptionChanged* handler.
 
 	public handler OnOptionChanged(in pIndex)
 		set property "timeZone" of mClock to pIndex - 1
 	end handler
 
+### Using native APIs for libraries
+
+One of the most powerful features of LiveCode Builder is the foreign 
+function interface (FFI). It can be used to access native APIs on all 
+supported platforms.
+
+The fundamental concept involved in using the FFI is the foreign handler 
+binding. This is a string containing information about the native 
+functionality that is being bound to.
+
+The format of the binding string varies slightly depending on the 
+language of the foreign handler - for a complete specification, see the
+LiveCode Builder Language Reference.
+
+#### C
+
+In order to bind to C functions from a dynamic library, there is one 
+essential thing to know: the signature of the function. 
+
+>**Note:** On Windows, it is also necessary to find out the calling 
+> convention of the function. In the majority of cases if binding to 
+> functions in the Windows API, the calling convention will be 
+> `stdcall`.
+
+The declared types of the foreign handler must match the signature, so
+that the FFI call can be computed correctly. For example, to bind to the 
+`atof` C standard library function that converts a C-string to a double,
+
+	double atof(const char *str)
```

Comment:
I think the `*` here has caused the text after that to be displayed in italics, until the next `*`

Reply:
Yeah, I think it's just a problem with the diff viewer - if you look at it here: https://github.com/livecode/livecode-ide/blob/1666cc79a403c91c8642fdaa4f527eb48f0533e7/Documentation/guides/Extending%20LiveCode.md
it doesn't have that effect

Reply:
(since it is in a code block it won't apply markup)